### ISSUE TITLE
Naming scheme of the Processor Setting?

### ISSUE THREAD

**Issue by yiyin** [FlyBrainLab/NeuroMynerva#32]
For hemibrain, only the one with `adult(hemibrain)` works anything else does not seem to work. 
This prohibits the use of two hemibrain settings each connecting to a different processor.

**Comment by TK-21st** [comment]
If you are referring to the 3D visualization settings and meshes then this is related to #20.

There needs to be a way for user to specify 3D visualization settings associated with each processor setting. 
If either one of you (@yiyin @mkturkcan) have some kind of updated schema for processor setting implemented in NeuroArch (or some other sources) I can update the front-end accordingly

https://github.com/FlyBrainLab/NeuroMynerva/issues/20#issuecomment-665828797

**Comment by yiyin** [comment]
No, I meant this file https://github.com/FlyBrainLab/NeuroMynerva/blob/master/schema/plugin.json.
The name has to be adult(hemibrain), adult(flycircuit), larva(l1em). If I have another hemibrain database that uses any name other than adult(hemibrain), the neuropils are not loaded properly.

**Comment by TK-21st** [comment]
It's the same issue. Currently when processor is set in `neu3dwidget`, the exact name of the processor is used to show meshes that are distributed with NeuroMynerva.

https://github.com/FlyBrainLab/NeuroMynerva/blob/ff7cbb72bf07b3506c950c020bf995645c2d8a49/src/widgets/neu3d-widget/widget.ts#L800-L867

**Comment by TK-21st** [comment]
A temporary solution is added in 4575723.
The neu3d visualization setting is now part of the schema. 

The user can do the following:
1. if one of the presets is specified, the default behavior for neu3d setting and meshes are included: (`larva(l1em)`,` adult(flycircuit)`,` hemibrain(flycircuit)`
2. otherwise, no meshes will be shown, and the camera setting is according to the user setting in the schema
